cms/models.py

```diff
@@ -74,15 +75,11 @@ def get_context(self, request, *args, **kwargs):
             **super().get_context(request, *args, **kwargs),
             "js_settings_json": json.dumps(_serialize_js_settings(request)),
             "title": self.title,
+            "site_name": settings.SITE_NAME,
             # The context variables below are added to avoid duplicate queries within the templates
             "three_column_image_text_section": self.three_column_image_text_section,
         }
 
-    def _get_child_page_of_type(self, cls):
-        """Gets the first child page of the given type if it exists"""
-        child = self.get_children().type(cls).live().first()
-        return child.specific if child else None
-
```

Comment:
Any reason why this was removed?

Reply:
Just a duplicate function. It's repeated just above